kristan/src/main/java/com/example/springboot/model/Program.java

```diff
@@ -0,0 +1,14 @@
+package com.example.springboot.model;
+
+import lombok.*;
+
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+
+public class Program {
+    private String programName;
+    private String programId;
+    private String programDesc;
+}
```

Comment:
Good Day, again thanks for introducing this lombok method to me, this save lots of typing and makes your code clean.